in_progress/mission-statement.md

```diff
@@ -0,0 +1,53 @@
+* Transparency -- we do our work as openly as possible and welcome our community to join in the decision making process
+* Simplicity – Invent as little as possible and work with standard pydata types. We strive for easy-to-use interfaces that reduce mental overhead.
+
+## Abstract
+
+The growing adoption of Python for single-cell omics data analysis has been catalyzed by [Scanpy](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1382-0) (single-cell analysis in Python) and its ecosystem.
+This ecosystem has outgrown a single package or modality and now routinely includes packages at the frontier of single-cell research and development. To handle the growing needs of this ecosystem, we've formed a new organization -- **`scverse`**.
+
+`Scverse` is a community-based ecosystem of tools with users and developers across many organizations.
+To sustain the utility and growth of the ecosystem, it’s essential that the core analytic tools are robust and well-maintained. Accomplishing this entails consistently supporting and improving tools beyond what can be done in the conventional single-lab academic setting.
+
+Scverse constitutes an environment of tools which is deliberately not unified under a single package, but instead is a consortium of core analytic tools with shared maintenance responsibilities.
+Tools within this environment that are considered "core" for sharing data between tools or are required for fundamental analyses of single cell datatypes are placed under shared maintainance in the `scverse` github organization.
+
+Workflows in this environment are heterogeneous and this core can be supplemented with analysis specific accessory packages.
+This document lays out our vision and processes around the project's community in a transparent and accessible way.
+We are committed to keeping this community open to third-party contributions and usage and are committed to acknowledging contributions.
+This white paper may serve as an entry point for third parties into the structure of the scverse environment.
+
+## User engagement
+
+The ecosystem centred on `scverse` core packages consists of python packages which are documented on readthedocs and commonly accompanied by scientific publications.
+Its primary purpose is to allow users to generalise workflows to their own data analysis settings.
+We promote user engagement via the following mechanisms:
+
+* Clear documentation of code and usage of tutorial notebooks.
+* Participation in development though issues and pull requests.
+* Community forums, such as our Discourse and Twitter account. scverse environment-centric workshops at which scverse workflows are taught and hackathons at which functionalities are contributed. We share event dates and resources within the environment to increase impact and to exploit synergies.
```

Comment:
```suggestion
* clear documentation of code and usage of tutorial notebooks,
* participation in development through issues and pull requests,
* community forums, such as our Discourse and Twitter account, 
* workshops at which we teach analysis workflows based on the `scverse` ecosystem, and
* hackathons, to lower the bar for code-contributions. 
```

> We share event dates and resources within the environment to increase impact and to exploit synergies.

"resources" and "environment" are quite unspecific, I guess we can as well delete that sentence.